processor/groupbytraceprocessor/processor.go

```diff
@@ -33,7 +36,7 @@ import (
 // Each in-flight trace is registered with a go routine, which will be called after the given duration and dispatched to the event
 // machine for further processing.
 // The typical data flow looks like this:
-// ConsumeTraces -> event(traceReceived) -> onBatchReceived -> AfterFunc(duration, event(traceExpired)) -> onTraceExpired
+// ConsumeTraces -> event(traceReceived) -> onTraceReceived -> AfterFunc(duration, event(traceExpired)) -> onTraceExpired
```

Comment:
Perhaps update the doc here to mention `consume(trace)` instead? Also mention that the batch is split per trace, and that traces are routed to specific workers based on the trace ID